timescaledb/tutorials/grafana/visualizations/pie-chart.md

```diff
@@ -0,0 +1,167 @@
+
+A donut chart is a pie chart with the middle cut out. Compared to a pie chart,
+it can be easier to compare segments on a donut chart because the human eye
+is better at comparing the lengths of arcs. Otherwise, they're exactly the same.
+
+Follow this section to create a donut chart displaying the average transaction
+volume of each stock within a bucket interval. The transaction volume per
+bucket is calculated from the daily cumulative traded volume, which is available
+in the `stocks_real_time` hypertable.
+
+<procedure>
+
+### Creating donut chart with volume transactions
+
+1.  At the psql prompt, update the earlier query to find the maximum `day_volume`
+   value for a symbol within a bucket. Then, subtract each maximum from the
+   previous bucket's maximum. The difference gives the traded volume for that
+   bucket:
+
+   ```sql
+      SELECT
+          time_bucket('$bucket_interval', time) AS time,
+          symbol,
+          MAX(day_volume) - LAG(MAX(day_volume), 1) OVER(
+              PARTITION BY symbol
+              ORDER BY time_bucket('$bucket_interval', time)
+          ) AS bucket_volume
+      FROM stocks_real_time srt
+      WHERE symbol = $symbol
+          AND time >= $__timeFrom()::timestamptz AND time < $__timeTo()::timestamptz
+      GROUP BY time_bucket('$bucket_interval', time), symbol;
+      ORDER BY time_bucket('$bucket_interval', time), symbol;
+      ```
+
+1.  In the Grafana dashboard, convert your pie chart to a donut
+   chart. In the symbol drop-down menu, select all the stocks you want to
+   compare. On the right side of the panel, click the pie chart drop-down. In
+   the `piechart type` field, select `donut`:
+
+  <img class="main-content__illustration" src="https://s3.amazonaws.com/assets.timescale.com/docs/images/tutorials/visualizations/piechart/piecharttype.png" alt="Screenshot of Grafana dashboard, showing pie chart."/>
+
+1.  Refresh the panel. The donut chart view shows the percentage of trading volume 
+    for a 10-minute bucket, averaged over the entire day:
+
+   <img class="main-content__illustration" src="https://s3.amazonaws.com/assets.timescale.com/docs/images/tutorials/visualizations/piechart/donutchart.png" alt="Screenshot of Grafana dashboard, showing a donut chart."/>
+
+
+<highlight type="note">
+If you go beyond a single trading day, you might get results that don't look
+very good, or you might get no data returned. To fix this, focus your
+calculation on a single trading day instead.
+</highlight>
+
+</procedure>
+
+Pie charts are a great tool for comparing categorized data. They're especially good
+for visualizing percentages. But they don't work as well if you have too many categories
+with similar percentages. With more slices, the chart becomes harder to analyze, and
+with very similar, small slices, it becomes harder to compare slice sizes. Use
+pie charts for a small number of categories, and consider another chart type when
+you have large amounts of data.
```

Comment:
I feel like this probably belongs at the top, as well.

Reply:
Sure

Reply:
Done